components/camel-kubernetes/pom.xml

```diff
@@ -47,6 +47,15 @@
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-cluster</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-util-json</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20200518</version>
```

Comment:
Please avoid using org.json:json, the license is not complaint with ASF, remove this.

Reply:
I'll change not to use this package.